projects/data-atlas-app/src/app/accordion-entry/accordion-entry.component.html

```diff
@@ -1,22 +1,24 @@
-<mat-card *ngFor="let dataset of datasets$ | async; index as i" class="dataset">
+<mat-card *ngFor="let collection of collections$ | async; index as i" class="dataset">
     <mat-card-header>
         <mat-icon mat-card-avatar>{{ icon }}</mat-icon>
-        <mat-card-title>{{ dataset.name }}</mat-card-title>
-        <mat-card-subtitle> {{ dataset.properties[0] ? dataset.properties[0][1] : 'no author information available' }}</mat-card-subtitle>
+        <mat-card-title>{{ collection.name }}</mat-card-title>
+        <mat-card-subtitle>
+            {{ collection.properties[0] ? collection.properties[0][1] : 'no author information available' }}</mat-card-subtitle
```

Comment:
Do we always have property 0 and 1?
Why is accessing this so "ugly"?

Reply:
Initially, I designed the properties to be a map. But then they have no order, and for the EBV portal we always want to show the authors first and the license second but it changed all the time. So I changed it to an array of key-value tuples.

Reply:
But isn't it strange that we assume that property 0 is the author?
Shouldn't we lookup the properties for the author?

Reply:
the netcdfcf layer provider only puts a single property in and that's the author. So no need to search for it.